nowcast/delphi_nowcast/sensorization/get_epidata.py

```diff
@@ -0,0 +1,131 @@
+import asyncio
+from datetime import datetime, date
+from typing import Tuple, List, Dict
+from itertools import product
+
+from numpy import isnan
+from pandas import date_range
+from aiohttp import ClientSession
+
+# from ..delphi_epidata import Epidata  # used for local testing
+from delphi_epidata import Epidata
+
+from ..data_containers import LocationSeries, SensorConfig
+
+
+async def get(params, session, sensor, location):
+    """Helper function to make Epidata GET requests."""
+    async with session.get(Epidata.BASE_URL, params=params) as response:
+        return await response.json(), sensor, location
+
+
+async def fetch_epidata(combos, as_of):
+    """Helper function to asynchronously make and aggregate Epidata GET requests."""
+    tasks = []
+    async with ClientSession() as session:
+        for sensor, location in combos:
+            params = {
+                    "endpoint": "covidcast",
+                    "data_source": sensor.source,
+                    "signals": sensor.signal,
+                    "time_type": "day",
+                    "geo_type": location.geo_type,
+                    "time_values": f"20200101-{as_of}",
+                    "geo_value": location.geo_value,
+                    "as_of": as_of
+                }
+            task = asyncio.ensure_future(get(params, session, sensor, location))
```

Comment:
`create_task()` is preferred to `ensure_future()`
https://docs.python.org/3/library/asyncio-future.html#asyncio.ensure_future

Reply:
oh cool, thanks for the info. Updated